### DISCUSSION TITLE
Waht do the recipe ingredient checkboxes do ?

### DISCUSSION THREAD

**Discussion by faspina** [TandoorRecipes/recipes#441] [Q&A]
What do the check boxes do beside each ingredient when you drill into a recipe. I checked a few items I had, uncheck items i needed and clicked add to list but all items from the recipe were put on the list.

**Comment by vabene1111** [comment]
they are just visual markers for when you are cooking and want to check the ingredients you already put in.

Avoids forgetting something if you have a large number of ingredients

**Comment by faspina** [comment]
How do you add only specific items from a recipe that you need without adding the whole list?

**Reply by vabene1111** [reply]
do you mean in the shopping list ? You can mark certain foods as excluded from shopping (by going to utilities > ingredients)
